### PR TITLE
Add information manipulation terms definitions in English

### PR DESCRIPTION
**Pull request by SiegridHenry**

### CODE REVIEW

File: _pages/en/encyclopedia/definitions [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+# Definitions
+
+The subject is riddled with an abundance of terms, mixing different notions and realities. 
+Below you will find definitions of the most commonly used terms and a precise definition of information manipulation.
+
+#### Fake news 
+
+is the most commonly used expression and does not necessarily refer to false news but rather falsified, counterfeit, or forged information.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:15]: I'm not sure of the readability of starting a paragraph as if it was the continuation of a sentence started by the title 🙂 
```suggestion
_Fake news_ is the most commonly used expression and does not necessarily refer to false news but rather falsified, counterfeit, or forged information. 
```

File: _pages/en/encyclopedia/definitions [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+# Definitions
+
+The subject is riddled with an abundance of terms, mixing different notions and realities. 
+Below you will find definitions of the most commonly used terms and a precise definition of information manipulation.
+
+#### Fake news 
+
+is the most commonly used expression and does not necessarily refer to false news but rather falsified, counterfeit, or forged information. 
+However, this term is vague and does not account for the fact that part of the problem arises from information that is not, strictly speaking, “false.”
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:16]: This seems inconsistent with what we've just said above: “does not necessarily refer to false news” 🤔 

```suggestion
This term is vague and does not account for the fact that part of the problem arises from information that is not, strictly speaking, “false.” 
```

File: _pages/en/encyclopedia/definitions [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# Definitions
+
+The subject is riddled with an abundance of terms, mixing different notions and realities. 
+Below you will find definitions of the most commonly used terms and a precise definition of information manipulation.
+
+#### Fake news 
+
+is the most commonly used expression and does not necessarily refer to false news but rather falsified, counterfeit, or forged information. 
+However, this term is vague and does not account for the fact that part of the problem arises from information that is not, strictly speaking, “false.” 
+The term has also become so over-used that it is not uncommon to see it for all news that we dislike, ultimately enabling a form of populism that is hostile to the freedom of the press.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:17]:
```suggestion
The term has also become so over-used that it is not uncommon to see it applied to all news that one dislikes, driving towards a generalized distrust in media.
```

File: _pages/en/encyclopedia/definitions [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+Source:[CAPS report](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/manipulations-de-l-information/rapport-conjoint-caps-irsem-les-manipulations-de-l-information-un-defi-pour-nos/)
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:18]:
```suggestion
Source: [CAPS report](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/manipulations-de-l-information/rapport-conjoint-caps-irsem-les-manipulations-de-l-information-un-defi-pour-nos/)
```

File: _pages/en/encyclopedia/definitions [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+#### Misinformation
+
+is the unintentional dissemination of information that is wholly or partly false or information whose inaccuracy is unintentional.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:22]:
```suggestion
_Misinformation_ is the unintentional dissemination of information that is wholly or partly false, or information for which inaccuracy is unintentional.
```

File: _pages/en/encyclopedia/definitions [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+Sources:[CAPS report](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/manipulations-de-l-information/rapport-conjoint-caps-irsem-les-manipulations-de-l-information-un-defi-pour-nos/) ; [Data&Society report](https://datasociety.net/pubs/oh/DataAndSociety_LexiconofLies.pdf)
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:23]: No space before `;`, `:`, `?`, `!` in English 😉 

```suggestion
Sources: [CAPS report](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/manipulations-de-l-information/rapport-conjoint-caps-irsem-les-manipulations-de-l-information-un-defi-pour-nos/); [Data & Society report](https://datasociety.net/pubs/oh/DataAndSociety_LexiconofLies.pdf)
```

File: _pages/en/encyclopedia/definitions [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+#### Malinformation
+
+is when genuine information is shared to cause harm, often by moving information designed to stay private into the public sphere.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:27]:
```suggestion
_Malinformation_ is when genuine information is shared to cause harm, often by moving information designed to stay private into the public sphere.
```

File: _pages/en/encyclopedia/definitions [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+Source: [Council of Europe report DGI(2017)09](https://rm.coe.int/information-disorder-report-november-2017/1680764666)
+
+#### Disinformation
+
+is the intentional dissemination of information that is wholly or partly false. It differs from “misinformation,” which is unintentional.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:32]:
```suggestion
_Disinformation_ is the intentional dissemination of information that is wholly or partly false. Only the active intention separates it from [misinformation](#Misinformation).
```

File: _pages/en/encyclopedia/definitions [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+#### Malinformation
+
+is when genuine information is shared to cause harm, often by moving information designed to stay private into the public sphere.
+Source: [Council of Europe report DGI(2017)09](https://rm.coe.int/information-disorder-report-november-2017/1680764666)
+
+#### Disinformation
+
+is the intentional dissemination of information that is wholly or partly false. It differs from “misinformation,” which is unintentional. 
+This definition is both too broad and too narrow. It is too broad in the sense that it includes benign information that lacks a hostile intention.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:33]: Mixing definition and judgement doesn't seem very appropriate to me. Let's keep the judgement in its own block, or perhaps even page, probably as “French position” 🙂

File: _pages/en/encyclopedia/definitions [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+#### Malinformation
+
+is when genuine information is shared to cause harm, often by moving information designed to stay private into the public sphere.
+Source: [Council of Europe report DGI(2017)09](https://rm.coe.int/information-disorder-report-november-2017/1680764666)
+
+#### Disinformation
+
+is the intentional dissemination of information that is wholly or partly false. It differs from “misinformation,” which is unintentional. 
+This definition is both too broad and too narrow. It is too broad in the sense that it includes benign information that lacks a hostile intention. 
+At the same time, the concept of disinformation is too narrow because the problems that we encounter are not all, strictly speaking, forms of disinformation.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:34]: It is not the _concept of disinformation_ itself that is too narrow (for what?): it is _the use of this narrow term as a generic way to designate the societal problem_ that is inappropriate. Let's stick to definitions here 🙂

File: _pages/en/encyclopedia/definitions [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+is when genuine information is shared to cause harm, often by moving information designed to stay private into the public sphere.
+Source: [Council of Europe report DGI(2017)09](https://rm.coe.int/information-disorder-report-november-2017/1680764666)
+
+#### Disinformation
+
+is the intentional dissemination of information that is wholly or partly false. It differs from “misinformation,” which is unintentional. 
+This definition is both too broad and too narrow. It is too broad in the sense that it includes benign information that lacks a hostile intention. 
+At the same time, the concept of disinformation is too narrow because the problems that we encounter are not all, strictly speaking, forms of disinformation. 
+Sometimes information is not false, but simply exaggerated, biased or presented in a very emotional way, such as in a tabloid.
+Information can be manipulated in many ways: through the production, dissemination and even retention. These processes do not all imply a dichotomy between truth and falsehood.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:36]: Not very clear.

File: _pages/en/encyclopedia/definitions [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+Source: [Council of Europe report DGI(2017)09](https://rm.coe.int/information-disorder-report-november-2017/1680764666)
+
+#### Disinformation
+
+is the intentional dissemination of information that is wholly or partly false. It differs from “misinformation,” which is unintentional. 
+This definition is both too broad and too narrow. It is too broad in the sense that it includes benign information that lacks a hostile intention. 
+At the same time, the concept of disinformation is too narrow because the problems that we encounter are not all, strictly speaking, forms of disinformation. 
+Sometimes information is not false, but simply exaggerated, biased or presented in a very emotional way, such as in a tabloid.
+Information can be manipulated in many ways: through the production, dissemination and even retention. These processes do not all imply a dichotomy between truth and falsehood. 
+Most of the time, the manipulator does not position himself relative to the truth; he or she is simply trying to produce an effect.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:37]: Use plural for gender neutrality 🙂

File: _pages/en/encyclopedia/definitions [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+This definition is both too broad and too narrow. It is too broad in the sense that it includes benign information that lacks a hostile intention. 
+At the same time, the concept of disinformation is too narrow because the problems that we encounter are not all, strictly speaking, forms of disinformation. 
+Sometimes information is not false, but simply exaggerated, biased or presented in a very emotional way, such as in a tabloid.
+Information can be manipulated in many ways: through the production, dissemination and even retention. These processes do not all imply a dichotomy between truth and falsehood. 
+Most of the time, the manipulator does not position himself relative to the truth; he or she is simply trying to produce an effect. 
+Source:[CAPS report](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/manipulations-de-l-information/rapport-conjoint-caps-irsem-les-manipulations-de-l-information-un-defi-pour-nos/)
+
+#### Information manipulation
+
+this term is more inclusive. Manipulation is intentional (its purpose is to cause harm) and clandestine (victims do not know they are being manipulated).
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:42]: The second sentence is super interesting, but this is not a definition 😉

File: _pages/en/encyclopedia/definitions [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+Most of the time, the manipulator does not position himself relative to the truth; he or she is simply trying to produce an effect. 
+Source:[CAPS report](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/manipulations-de-l-information/rapport-conjoint-caps-irsem-les-manipulations-de-l-information-un-defi-pour-nos/)
+
+#### Information manipulation
+
+this term is more inclusive. Manipulation is intentional (its purpose is to cause harm) and clandestine (victims do not know they are being manipulated). 
+France uses this term encompassing [specific criteria](https://disinfo.quaidorsay.fr/encyclopedia/france): a coordinated campaign, the diffusion of false information or information that is consciously distorted, and the political intention to cause harm. 
+The notion of a “coordinated campaign” refers less to the idea of an orchestrated operation with certain actors giving orders and others carrying them out. 
+It refers more to a range of indicators pointing to the diffusion, through various media, of problematic content, by both human and non-human sources.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:45]: “coordinated campaign“ should probably have its own definition then.

File: _pages/en/encyclopedia/definitions [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+Source: [CAPS report](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/manipulations-de-l-information/rapport-conjoint-caps-irsem-les-manipulations-de-l-information-un-defi-pour-nos/)
+
+## Taxonomy
+
+This table is not intended to be exhaustive, but to provide a key to understanding the most characteristic phenomena for each of the terms
+(inspired by [James Pamment, Carnegie Endowment for International Peace](http://carnegieendowment.org/files/2020-How_do_you_define_a_problem_like_influence.pdf)).
+
+|:---------| Misinformation  | Disinformation     | Information manipulation |
+| :------- |:---------------:| ------------------:|-------------------------:|
+| Actor(s) | Any, usually an isolated actor  |   Any       |  Usually an organisation |
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:55]:
```suggestion
| Actor(s) | Any, usually a single actor  |   Any       |  Usually an organisation |
```

File: _pages/en/encyclopedia/definitions [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+| Behavior(s) | Unintentional dissemination| Intentional dissemination with the aim of producing an effect  | An intentional and coordinated diffusion with a political intention to cause harm  |
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:56]: So there cannot be non-coordinated “information manipulation”? 🤔

File: _pages/en/encyclopedia/definitions [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+
+#### Information manipulation
+
+this term is more inclusive. Manipulation is intentional (its purpose is to cause harm) and clandestine (victims do not know they are being manipulated). 
+France uses this term encompassing [specific criteria](https://disinfo.quaidorsay.fr/encyclopedia/france): a coordinated campaign, the diffusion of false information or information that is consciously distorted, and the political intention to cause harm. 
+The notion of a “coordinated campaign” refers less to the idea of an orchestrated operation with certain actors giving orders and others carrying them out. 
+It refers more to a range of indicators pointing to the diffusion, through various media, of problematic content, by both human and non-human sources.
+Source: [CAPS report](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/manipulations-de-l-information/rapport-conjoint-caps-irsem-les-manipulations-de-l-information-un-defi-pour-nos/)
+
+## Taxonomy
```
Comment (MattiSG) [inline _pages/en/encyclopedia/definitions:48]: If it is not intended to be exhaustive, I don't think we can call it a taxonomy 😉

**Comment by MattiSG** [comment]
Additional reference: brand new [report from OECD](https://read.oecd-ilibrary.org/view/?ref=135_135220-cvba4lq3ru&title=Transparency-communication-and-trust-The-role-of-public-communication-in-responding-to-the-wave-of-disinformation-about-the-new-coronavirus) ([French version](https://read.oecd-ilibrary.org/view/?ref=135_135223-duu3s3o3df&title=Transparence-communication-et-confiance-Le-role-de-la-communication-publique-pour-combattre-la-vague-de-desinformation-concernant-le-nouveau-coronavirus)).

**Comment by MattiSG** [comment]
@Ndpnt can you please deploy this if tests pass? Thanks! :)

**Comment by Ndpnt** [comment]
> @Ndpnt can you please deploy this if tests pass? Thanks! :)

Yep, I'm on it. 
I just have to install ruby 2.7.1… 😅

**Pull request merged**